### PR TITLE
Rename `qiskit_superstaq.IICCX` to `qiskit_superstaq.AQTICCX`

### PR DESCRIPTION
**Pull request by vtomole**

### CODE REVIEW

File: qiskit_superstaq/__init__.py [hunk 1]
```diff
@@ -6,11 +6,11 @@
 from ._version import __version__
 from .custom_gates import (
     AceCR,
-    ICCXdgGate,
-    ICCXGate,
-    IICCX,
-    IITOFFOLI,
-    ITOFFOLIGate,
+    AQTiCCX,
```
Comment (richrines1) [inline qiskit_superstaq/__init__.py:9]: do we need this one? qiskit generally doesn't have gate class names that don't end in "Gate", and we don't do the same for `ZZSwapGate`

File: qiskit_superstaq/__init__.py [hunk 2]
```diff
@@ -6,11 +6,11 @@
...
+    AQTiCCXdgGate,
+    AQTiCCXGate,
+    AQTiToffoli,
```
Comment (richrines1) [inline qiskit_superstaq/__init__.py:12]: ditto

**Comment by richrines1** [comment]
does this subsume #151?

**Comment by richrines1** [comment]
does this subsume #151?

**Comment by vtomole** [comment]
Oh lawwd, i forgot that PR exists. Let's use that one. I've pushed the recent changes to it.

**Pull request closed without merging**